resources/views/actions/create.blade.php

```diff
@@ -0,0 +1,66 @@
+@extends('layouts.master')
+
+@section('main_content')
+
+    @include('layouts.header', ['header' => 'Action Creation'])
+
+    <div class="container -padded">
+        <div class="wrapper">
+            <div class="container__block -narrow">
+                <h1>Add Post Metadata</h1>
+                <form method="post" enctype="application/x-www-form-urlencoded" action="{{ route('actions.store', ['campaign_id' => $campaignId]) }}">
+                {{ csrf_field()}}
+
+                    <div class="form-item">
+                        <label class="field-label">Action Name</label>
+                        <input type="text" name="name" class="text-field" placeholder="Name your action e.g. Teens for Jeans Photo Upload" value="{{old('internal_title') }}">
+                    </div>
```

Comment:
@ngjo @lkpttn is the below naming convention finalized? If so, I can add as the placeholder instead of "Name your action..."
```
CampaignAbbrev- YYYY Action Description
Example: SSD-2019 Bin
```